### DISCUSSION TITLE
Unable to create new Project

### DISCUSSION THREAD

**Discussion by gunpal5** [abpframework/abp#6998] [General]
Hello,

I've been trying to create new ABP project but ABP CLI is failing to do so. I believe there's some issue at server side.

![image](https://user-images.githubusercontent.com/10114874/103537488-83612f00-4eba-11eb-99ce-a874e605d65e.png)

![image](https://user-images.githubusercontent.com/10114874/103537644-c6bb9d80-4eba-11eb-924c-fbb2e8d0afdb.png)

Forgive me If this is not the right place to report this issue. I don't know better place to report this bug.

Regards,
Gunpal Jain

**Comment by maliming** [comment]
Hi

We will fix it asap.

**Reply by gunpal5** [reply]
Thank you for quick response

**Reply by maliming** [reply]
it works now.

**Reply by gunpal5** [reply]
Thank you!

**Reply by Gitvazonline** [reply]
This error is back

**Reply by maliming** [reply]
We will fix it as soon as possible.

**Comment by 274188A** [comment]
Still getting this just now for 4.1.1Issue here: https://github.com/abpframework/abp/issues/7202
